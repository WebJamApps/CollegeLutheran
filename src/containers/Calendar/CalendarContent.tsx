import React from 'react';

const CalendarContent = (): JSX.Element => (
  <div className="page-content">
    <div className="container-fluid">
      <p style={{ fontSize: '4pt', margin: '0' }}>&nbsp;</p>
      <div className="material-content elevation3" style={{ maxWidth: '998px', paddingBottom: '-80px', margin: 'auto' }}>
        <iframe
          src="https://calendar.google.com/calendar/embed?title=College%20Lutheran%20Events%20Calendar&amp;showTabs=0&amp;showCalendars=0&amp;
          height=900&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=ra8um2u26p59a2t92lfooq4els%40group.calendar.google.com&amp;
          color=%23060D5E&amp;ctz=America%2FNew_York"
          style={{ borderWidth: '0' }}
          width="900"
          height="900"
          frameBorder="0"
          scrolling="no"
          title="Church Calendar"
        />
      </div>
    </div>
  </div>
);

export default CalendarContent;
