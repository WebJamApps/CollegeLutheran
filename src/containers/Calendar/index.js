import React, { Component } from 'react';
import DefaultCalendarContent from './CalendarContent';
import commonUtils from '../../lib/commonUtils';

export default class Calendar extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Church Calendar'); }

  render() { return (<DefaultCalendarContent />); }
}
