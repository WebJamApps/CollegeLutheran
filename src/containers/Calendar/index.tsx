import React, { Component } from 'react';
import DefaultCalendarContent from './CalendarContent';
import commonUtils from '../../lib/commonUtils';

export default class Calendar extends Component {
  commonUtils: {setTitleAndScroll: (pageTitle: string, width: number) => void;};

  constructor(props: Readonly<Calendar>) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Church Calendar', window.screen.width); }

  render(): JSX.Element { return (<DefaultCalendarContent />); }
}
