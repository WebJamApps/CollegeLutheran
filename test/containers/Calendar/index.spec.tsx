import Calendar from 'src/containers/Calendar';
import renderer from 'react-test-renderer';

describe('Calendar', () => {
  it('renders correctly', () => {
    const c = renderer.create(<Calendar />).toJSON();
    expect(c).toMatchSnapshot();
  });
});
