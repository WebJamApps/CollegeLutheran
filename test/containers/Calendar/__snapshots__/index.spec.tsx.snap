// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar renders correctly 1`] = `
<div
  className="page-content"
>
  <div
    className="container-fluid"
  >
    <p
      style={
        {
          "fontSize": "4pt",
          "margin": "0",
        }
      }
    >
      Â 
    </p>
    <div
      className="material-content elevation3"
      style={
        {
          "margin": "auto",
          "maxWidth": "998px",
          "paddingBottom": "-80px",
        }
      }
    >
      <iframe
        frameBorder="0"
        height="900"
        scrolling="no"
        src="https://calendar.google.com/calendar/embed?title=College%20Lutheran%20Events%20Calendar&showTabs=0&showCalendars=0&
          height=900&wkst=1&bgcolor=%23FFFFFF&src=ra8um2u26p59a2t92lfooq4els%40group.calendar.google.com&
          color=%23060D5E&ctz=America%2FNew_York"
        style={
          {
            "borderWidth": "0",
          }
        }
        title="Church Calendar"
        width="900"
      />
    </div>
  </div>
</div>
`;
